testsuite/features/step_definitions/cobbler_steps.rb

```diff
@@ -233,6 +233,7 @@
   $server.run(cleanup_command.to_s, check_errors: false)
   result, code = $server.run('systemctl restart apache && systemctl restart cobblerd')
```

Comment:
If we restart apache and cobbler we can't be if the problem is on apache or cobbler.